randovania/games/super_metroid/game_data.py

```diff
@@ -3,6 +3,7 @@
 from randovania.games.super_metroid.layout.super_metroid_cosmetic_patches import SuperMetroidCosmeticPatches
 from randovania.games.game import GameData, GameGenerator, GameGui, GameLayout
 from randovania.games.super_metroid.patcher.super_duper_metroid_patcher import SuperDuperMetroidPatcher
+from randovania.games.super_metroid.gui.dialog.super_cosmetic_patches_dialog import SuperCosmeticPatchesDialog
```

Comment:
this import being outsidde the `*_gui()` function is what's causing the docker test to fail.